src/Lifi.ts

```diff
@@ -13,24 +14,43 @@ import {
   RoutesRequest,
   RoutesResponse,
   Step,
-  StepTransactionResponse,
   Token,
   TokenAmount,
+} from '@lifinance/types'
+
+import {
+  Config,
+  ConfigUpdate,
   ExecutionData,
   ActiveRouteDictionary,
   ExecutionSettings,
-  DefaultExecutionSettings,
 } from './types'
 
 class LIFI {
   private activeRoutes: ActiveRouteDictionary = {}
-  private config = {
-    apiUrl: process.env.REACT_APP_API_URL || 'https://test.li.finance/api/',
+
+  private config: Config = getDefaultConfig()
+
+  getConfig = () => {
+    return this.config
+  }
+
+  setConfig = (configUpdate: ConfigUpdate) => {
+    this.config = mergeConfig(this.config, configUpdate)
+    return this.config
   }
 
   getPossibilities = async (
     request?: PossibilitiesRequest
   ): Promise<PossibilitiesResponse> => {
+    if (!request) request = {}
+
+    // apply defaults
+    request.bridges = request.bridges || this.config.defaultRouteOptions.bridges
```

Comment:
shouldn't this be the intersection of the requested bridges from `request` and the bridges that where enabled in the `defaultRouteOptions`?

Reply:
It is easy to create a blocking combination if allowed and denied bridges. I thought it may be clearer for the user that if he defines such an object it is actually used. And not merged with another one internally.

Reply:
Hmm I agree but also strange if you define something in the configuration step that doesn't have any effect in the next step